src/components/Body.vue

```diff
@@ -4,21 +4,51 @@
     <p>New in Taiwan?<br />Here are some converters to help you out!<br /></p>
   </div>
   <div>
-    <Currency />
+    <Converter label="USD - NTD" outputLabel="NTD Equivalent" :output="usdConvert" />
+    <Converter label="NTD - USD" outputLabel="NTD Equivalent" :output="ntdConvert" />
+    <Converter label="MPH - KPH" outputLabel="KPH Equivalent" :output="mphConvert" />
+    <Converter label="KPH - MPH" outputLabel="KPH Equivalent" :output="kphConvert" />
+    <Converter label="F ° - C °" outputLabel="C ° Equivalent" :output="fConvert" />
+    <Converter label="C ° - F °" outputLabel="C ° Equivalent" :output="cConvert" />
   </div>
 </template>
 
 <script>
-import Currency from "./Currency.vue";
+import Converter from "./Converter.vue";
 
 export default {
   name: "Body",
   props: {
     msg: String,
-    convert: Function,
+    ntdOutput: Function,
   },
+  data() {
+    return {
+      usdConvert(convert) {
+        return Number(Math.round(convert * 27.65));
+      },
+      ntdConvert(convert) {
+        return Number((convert * 0.036).toFixed(2));
+      },
+      mphConvert(convert) {
+        return Number(Math.round(convert * 1.6 * 10) / 10);
+      },
+      kphConvert(convert) {
+        return Number(Math.round(convert * 0.62 * 10) / 10);
+      },
+      fConvert(convert) {
+        if(convert == 0) return Number(0);
```

Comment:
change to `===` 

https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/types%20%26%20grammar/ch4.md#loose-equals-vs-strict-equals

Reply:
ah, um, btw, you might find that if you change to `===`, it stops working, you'll probably have to change to `if(Number(convert) === 0)`, read the above link to find out why

Reply:
It did want `if(Number(convert) === 0)` in order to work properly, cool.